src/set_theory/ordinal/basic.lean

```diff
@@ -742,7 +742,7 @@ theorem card_le_card {o₁ o₂ : ordinal} : o₁ ≤ o₂ → card o₁ ≤ car
 induction_on o₁ $ λ α r _, induction_on o₂ $ λ β s _ ⟨⟨⟨f, _⟩, _⟩⟩, ⟨f⟩
 
 instance : has_zero ordinal :=
-⟨⟦⟨pempty, empty_relation, by apply_instance⟩⟧⟩
+⟨@type pempty empty_relation _⟩
```

Comment:
Perhaps to make this look less hacky
```suggestion
⟨type (empty_relation : pempty → pempty → Prop)⟩
```
Up to you

Reply:
I don't think this looks all that hacky. `0` is the type of the `empty_relation` on `pempty`, and that can be read immediately from the way I wrote it down.